jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/AuthParameters.java

```diff
@@ -24,14 +25,15 @@
  * A bean that configures authorization credentials to be used for a registry. This is configurable
  * with Groovy closures and can be validated when used as a task input.
  */
-public class AuthParameters {
+public class AuthParameters implements AuthProperty {
 
   @Nullable private String username;
   @Nullable private String password;
 
   @Input
-  @Nullable
   @Optional
+  @Override
```

Comment:
nit: Not specific to this PR, but I was thinking maybe we could follow a convention of alphabetical ordering for annotations just for consistency in the future - not important though.